.gitignore

```
@@ -4,5 +4,6 @@
 *.a
 *.pyc
 __pycache__
-sktools/build
-
+build
+_build
```

Only ignore those in the project root?
```suggestion
/build*
/_build*
```